README.md

```diff
@@ -1,10 +1,10 @@
-# Basic Site Template
+# Standard Site Template
 
-This is the basic site template for Adobe Experience Manager as a Cloud Service (AEMaaCS).
+This is the standard site template for Adobe Experience Manager as a Cloud Service (AEMaaCS).
 
-<img src="previews/site.png?raw=true" alt="Basic site preview" width="50%">
+<img src="previews/site.png?raw=true" alt="Standard site preview" width="50%">
```

Comment:
@comanV this image doesn't seem to display, shouldn't it instead be `previews/StandardSiteTemplate.png`?

Reply:
Yes you are right, it was changed to `previews/StandardSiteTemplate.png` here https://github.com/adobe/aem-site-template-standard/commit/270cc0329fa39e4a90874614724aa9d5690aca45.